sirepo/pkcli/admin.py

```diff
@@ -23,6 +24,10 @@
 import shutil
 
 
+_MILLISECONDS_PER_MONTH = 30.4167 * 24 * 60 * 60 * 1000
```

Comment:
This should not be 30.4167. It should be 30. That's sufficiently accurate for this purpose and keeps this number an int, which it should be.